conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ConjureJerseyFeature.java

```diff
@@ -17,23 +17,21 @@
 package com.palantir.conjure.java.server.jersey;
 
 import com.fasterxml.jackson.jaxrs.cbor.JacksonCBORProvider;
+import com.google.errorprone.annotations.CheckReturnValue;
 import com.palantir.conjure.java.serialization.ObjectMappers;
+import com.palantir.logsafe.Preconditions;
 import com.palantir.tracing.jersey.TraceEnrichingFilter;
-import com.palantir.tritium.metrics.registry.SharedTaggedMetricRegistries;
+import java.util.function.Consumer;
 import javax.ws.rs.core.Feature;
 import javax.ws.rs.core.FeatureContext;
 
 public enum ConjureJerseyFeature implements Feature {
     INSTANCE;
 
-    private final JerseyServerMetrics metrics;
+    private final Consumer<Throwable> listener;
 
     ConjureJerseyFeature() {
-        this.metrics = JerseyServerMetrics.of(SharedTaggedMetricRegistries.getSingleton());
-    }
-
-    public JerseyServerMetrics getMetrics() {
```

Comment:
I've deleted this method because I can't find any usages of it internally:

https://sourcegraph.p.b/search?q=ConjureJerseyFeature.INSTANCE.getMetrics%28%29+lang:java&patternType=literal